stubs/Flask-Cors/flask_cors/core.pyi

```diff
@@ -0,0 +1,66 @@
+from datetime import timedelta
+from logging import Logger
+from typing import Any, Iterable, Pattern, TypeVar, overload
+from typing_extensions import TypedDict
+
+_IterableT = TypeVar("_IterableT", bound=Iterable[Any])
+_T = TypeVar("_T")
+_App = Any  # flask is not part of typeshed
+_Response = Any  # flask is not part of typeshed
+_MultiDict = Any  # werkzeug is not part of typeshed
+_Options = TypedDict(
```

Comment:
You should use class syntax for TypedDict: `class _Options(TypedDict, tootal=False):`.